src/js/node/dgram.ts

```diff
@@ -53,7 +53,7 @@ const {
   validateAbortSignal,
 } = require("internal/validators");
 
-const { isIP } = require("./net");
+const { isIP } = require("node:net");
```

Comment:
we should move this into a separate function so we don't load all of node:net just for the isIP function. that can be done in a separate PR.